net/acme/files/run.sh

```diff
@@ -248,6 +246,7 @@ fi
 
 trap err_out HUP TERM
 trap int_out INT
+trap post_checks EXIT
```

Comment:
So this is a "just-in-case" thing? There are not actually any paths in issue_cert that don't call post_checks before returning...